src/containers/Review/ReviewHome.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { Message, Segment, Header } from 'semantic-ui-react'
+import { Loading } from '../../components'
+import strings from '../../utils/constants'
+import useGetFullApplicationStructure from '../../utils/hooks/useGetFullApplicationStructure'
+import { AssignmentDetailsNEW, FullStructure } from '../../utils/types'
+import ReviewSectionAssignment from './ReviewSectionAssignment'
+
+interface ReviewHomeProps {
+  assignments: AssignmentDetailsNEW[]
+  structure: FullStructure
+}
+
+const ReviewHome: React.FC<ReviewHomeProps> = ({ assignments, structure }) => {
+  const { error, fullStructure: fullApplicationStructure } = useGetFullApplicationStructure({
+    structure,
+    firstRunValidation: false,
+    shouldCalculateProgress: false,
+  })
```

Comment:
Why need to call this here (same thing in `ReviewPageWrapper`? I thought we were passing from `ApplicationWrapper` so we don't need to call here again. And you are passing the previous structure to it, this could be improved, will continue looking to suggest something.

Reply:
ApplicationWrapper doesn't call useGetFullApplicationStructure (just the static bits of structure that dont' change).

There are some diagrams linked to the issue and the PR.

We want evaluated elements (for visibility, is required etc..).

But don't want to re-evaluate them when review_responses change (which is how it was before this PR). I remember now why in the early diagrams i drew it this way.

This came in to light with the idea of getting reviewStructure for individual sections (it would mean for each section/review-assignment combination we would need to re-evaluate all elements in the application)